src/js/Physics.js

```diff
@@ -0,0 +1,41 @@
+import CANNON from 'cannon';
+
+const PHYSICS_DELTA = 1 / 60;
+const SUBSTEPS = 1;
+
+class Physics extends CANNON.World {
+
+  constructor() {
+    super();
+    this.broadphase = new CANNON.NaiveBroadphase();
```

Comment:
The `NaiveBroadphase` has a complexity of `N^2`. The `GridBroadphase` or `SAPBroadphase` may give better performance.